src/features/rewards/donationOverlay/style.ts

```diff
@@ -170,3 +171,25 @@ export const StyledClose = styled<{}, 'button'>('button')`
   color: #FFF;
   z-index: 2;
 `
+
+export const StyledLogoImage = styled<StyleProps, 'div'>('div')`
+  width: 148px;
```

Comment:
I think that size should be the same as the current letter

![image](https://user-images.githubusercontent.com/9574457/50325859-82355300-04e7-11e9-9506-6e534c0b161a.png)

vs

![image](https://user-images.githubusercontent.com/9574457/50325862-882b3400-04e7-11e9-86a2-8286c0344b1b.png)

Reply:
done